<launch>
    <!--- Sim Time -->
    <param name="/use_sim_time" value="true" />

    <!-- Parameters -->
    <rosparam file="$(find faster_lio_sam)/yaml/parameter.yaml" command="load" />

    <node pkg="faster_lio_sam" type="livox_repub" name="livox_repub" output="screen" />

    <!-- <node pkg="faster_lio_sam" type="combineMultiLivox" name="combineMultiLivox" output="screen" /> -->

    <node pkg="faster_lio_sam" type="ImageProjection" name="ImageProjection" output="screen" />

    <node pkg="faster_lio_sam" type="imuPreintegration" name="imuPreintegration" output="screen" />
    
    <node pkg="faster_lio_sam" type="fusionOptimization" name="fusionOptimization" output="screen" />

    <!-- <node pkg="faster_lio_sam" type="poseOptimization" name="poseOptimization" output="screen" /> -->

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find faster_lio_sam)/rviz/rviz.rviz" />
</launch>